datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts

```diff
@@ -11,23 +11,32 @@ function matchesTagsOrTerms(field: SchemaField, filterText: string, entityRegist
                 .getDisplayName(EntityType.GlossaryTerm, termAssociation.term)
                 .toLocaleLowerCase()
                 .includes(filterText),
-        )
+        ) ||
+        field.description?.toLocaleLowerCase().includes(filterText)
     );
 }
 
-// returns list of fieldPaths for fields that have Terms or Tags matching the filterText
+// returns list of fieldPaths for fields that have Terms or Tags or Descriptions matching the filterText
 function getFilteredFieldPathsByMetadata(editableSchemaMetadata: any, entityRegistry, filterText) {
     return (
         editableSchemaMetadata?.editableSchemaFieldInfo
-            .filter((fieldInfo) => matchesTagsOrTerms(fieldInfo, filterText, entityRegistry))
+            .filter((fieldInfo) => matchesTagsOrTermsOrDescription(fieldInfo, filterText, entityRegistry))
             .map((fieldInfo) => fieldInfo.fieldPath) || []
     );
 }
 
+function matchesDescription(fieldDescription: any, filterText: string) {
+    return fieldDescription?.toLocaleLowerCase().includes(filterText);
+}
+
 function matchesEditableTagsOrTerms(field: SchemaField, filteredFieldPathsByEditableMetadata: any) {
```

Comment:
now that `filteredFieldPathsByEditableMetadata` is also checking by description, you can rename this function `matchesEditableTagsOrTermsOrDescription`

Reply:
Fixed and committed!!